addons/mail/static/src/messaging/component/chatter/chatter.xml

```diff
@@ -40,6 +40,7 @@
                             hasComposer="false"
                             haveMessagesAuthorRedirect="true"
                             order="'desc'"
+                            scrollAdjust="false"
```

Comment:
Convention for boolean props to be prefixed with `is/are/has/have`, here `hasMessageListScrollAdjust`.